src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -180,41 +180,29 @@ static int client_ensure_duid(sd_dhcp6_client *client) {
         return dhcp_identifier_set_duid_en(&client->duid, &client->duid_len);
 }
 
-int sd_dhcp6_client_set_duid(
-                sd_dhcp6_client *client,
-                uint16_t type,
-                uint8_t *duid, size_t duid_len) {
+int sd_dhcp6_client_set_duid(sd_dhcp6_client *client, uint16_t duid_type,
+                             size_t duid_len, uint8_t *duid) {
```

Comment:
I renamed the parameters for more clarity. Looking more closely, it does fall out of line with the systemd pattern  - f(data* , size_t) vs. f(size_t , data *) .

I just have a habit of writing it the latter way. I'll fix it to read:
sd_dhcp6_client_set_duid(sd_dhcp6_client *client, uint16_t duid_type,
                                           uint8_t *duid, size_t duid_len)